InfiniLink/BLE/BLEWriteManager.swift

```diff
@@ -7,6 +7,7 @@
 
 import Foundation
 import CoreBluetooth
+import simd
```

Comment:
What is this import for? I don't see any calls to it and your PR builds fine without it.

Reply:
Looks like I must've left that there from when I was trying to get the time label to work, so that can be removed.